lms/envs/common.py

```diff
@@ -154,6 +154,9 @@
     # Don't autoplay videos for students
     'AUTOPLAY_VIDEOS': False,
 
+    # Whether to advance to next unit when a video ends, by default
+    'AUTOADVANCE_VIDEOS': False,
+
```

Comment:
The code using this feature flag is in common and will be executed in both the LMS and Studio. Because of this, please also set the default in `cms/envs/common.py` as well!